### ISSUE TITLE
java.lang.ArrayIndexOutOfBoundsException in slack bot

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#856]
Slack bot failed to answer for `hello`:

```
com.zerocracy.radars.slack.ReProfile[48] java.lang.ArrayIndexOutOfBoundsException: 1
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@g4s8 a typo, fixed, thanks: d9a1e3dee5491c86392a282956738f8b50455203

**Comment by 0crat** [comment]
The job is not in WBS, won't close the order
